pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -23,17 +27,31 @@ type executionStartInfo struct {
 	startTime time.Time
 	traceID   uint64
 	spanID    uint64
+	parentID  uint64
+}
```

Comment:
I think this makes sense for this PR, but in the future we will need to be able to extract from a lot of different JSON formats.